docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Getting Started with Pre-Aggregations
+permalink: /caching/pre-aggregations/getting-started
+category: Caching
+menuOrder: 2
+---
+
+Often at the beginning of an analytical system's lifecycle - when there is less
+data that queries execute over - the system works well and delivers responses
+within acceptable thresholds. However, as the size of the dataset grows, the
+time-to-response from a user's perspective can often suffer quite heavily. This
+is true of both consumer-grade and purpose-built data warehousing solutions.
+
+This leaves us with a chicken-and-egg problem; consumer-grade databases can
+deliver low-latency responses with small-to-large datasets, but struggle with
+massive analytical datasets; data warehousing solutions _usually_ make no
+guarantees except to deliver a response, which means latency can vary wildly on
+a query-to-query basis.
+
+| Database Type                   | Low Latency? | Massive Datasets? |
+| ------------------------------- | ------------ | ----------------- |
+| Consumer-grade (Postgres/MySQL) | ✅           | ❌                |
+| Analytical (BigQuery/Redshift)  | ❌           | ✅                |
+
+Cube.js provides a solution to this problem: pre-aggregations. In layman's
+terms, a pre-aggregation is a condensed version of the source data. It specifies
+attributes from the source, which Cube.js uses to condense (or crunch) the data.
+This simple yet powerful optimization can reduce the size of the dataset by
+several orders of magnitude, and ensures subsequent queries can be served by the
+same condensed dataset if any matching attributes are found.
+
+Pre-aggregations are defined within each cube's data schema, and cubes can have
+as many pre-aggregations as they require. The pre-aggregated data can be stored
+either alongside the source data in the same database, in an external database
+that is supported by Cube.js, or in Cube Store, a dedicated pre-aggregation
```

Comment:
Maybe add a link to the Cube Store documentation?

Reply:
Yep need to do another pass and add links, just wanted to get this draft out and get some eyes on it.